versioned/persist/nontx/src/main/java/org/projectnessie/versioned/persist/nontx/NonTransactionalOperationContext.java

```diff
@@ -15,11 +15,14 @@
  */
 package org.projectnessie.versioned.persist.nontx;
 
-public class NonTransactionalOperationContext {
+public class NonTransactionalOperationContext implements AutoCloseable {
```

Comment:
Is this needed for the key list fix? maybe a separate PR?

Reply:
Just for this PR, yes.